system-test/system-test-specs.js

```diff
@@ -89,17 +77,107 @@ describe('system-test (' + desired.browserName + ')', function() {
     after(function(done) {
         var result = browser
             .quit();
-        if (result.sauceJobStatus) {
-            result = result.sauceJobStatus(allPassed);
-        }
+        // if (result.sauceJobStatus) {
```

Comment:
@aluu317 why was this commended out?  was this causing the second test to fail?  if so, are we depending on these tests running in a certain order?